api/conf/conf.yaml

```diff
@@ -17,16 +17,18 @@
 
 conf:
   listen:
-    host: 127.0.0.1
-    port: 8080
+    host: 127.0.0.1          # `manager api` listening ip or host name
+    port: 8080               # `manager api` listening port
   etcd:
-    endpoints:
+    endpoints:               # it's possible to define multiple etcd hosts addresses of the same etcd cluster.
```

Comment:
please use better comments, it is not easy to understand

Reply:
I copy them from APISIX.

Reply:
we need to update the APISIX too ^_^

Reply:
`supports defining multiple etcd host addresses for an etcd cluster`

how about this?

Reply:
it's better.